### DISCUSSION TITLE
Interpretable models memory issues

### DISCUSSION THREAD

**Discussion by agt64** [autogluon/autogluon#2016] [General]
Hi everyone,

Great job with incorporating the imodels package! Very exciting to see. I have a dataset that's approx 1 million records with approx 100 features, and even with using a ml.m5.24xlarge (384 GB RAM) it's still requiring more memory. Is that expected with these models?

Thanks for the help and great work again all!

**Comment by Innixma** [comment]
You will want to provide the full log output so it is easier to assist.
